Tools/TestWebKitAPI/Tests/WTF/Packed.cpp

```diff
@@ -96,7 +96,7 @@ TEST(WTF_Packed, PackedAlignedPtr)
     }
     {
         PackedAlignedPtr<uint8_t, 16> key { nullptr };
-#if (OS(IOS) || OS(TVOS) || OS(WATCHOS)) && CPU(ARM64)
+#if (OS(IOS) || OS(TVOS) || OS(WATCHOS)) && CPU(ARM64) && !PLATFORM(IOS_FAMILY_SIMULATOR)
```

Comment:
This might need OS(VISION), as well. I assume you are not using IOS_FAMILY because this isn't the right size for MACCATALYST.